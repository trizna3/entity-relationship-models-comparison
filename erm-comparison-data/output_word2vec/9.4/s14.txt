vehicle-vehicles
vehicle type-EMPTY
line-line;routes
stop-stops
CHANGE_CARDINALITY:S-types-line
CHANGE_CARDINALITY:S-line-types
CHANGE_CARDINALITY:S-types-vehicles
CHANGE_CARDINALITY:S-vehicles-types
REBIND_1NN1_TO_MN:S-types
CONTRACT_11_ASSOCIATION:S-line-routes
