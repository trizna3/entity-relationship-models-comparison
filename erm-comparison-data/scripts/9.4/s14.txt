entity set stops {
	"name"
	"x"
	"y"
};

entity set line {
	"number"
};

entity set routes {
};

entity set times {
	"minutes"
};


entity set types {
	"type"
};

entity set vehicles {
	"numerical code"
};

entity set "leave time" {
	"hour"
	"minute"
};

association "duration" between {
	stops ONE
	line MANY
	times ONE
}{};

association between {
	stops MANY
	routes MANY
}{};

association between {
	line ONE
	routes ONE
}{};

association between {
	line MANY
	types ONE
}{};

association between {
	types ONE
	vehicles MANY
}{};

association between {
	vehicles ONE
	"leave time" MANY
}{};