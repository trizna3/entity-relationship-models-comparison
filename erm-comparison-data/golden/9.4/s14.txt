vehicle-vehicles
vehicle type-types
line-lines
stop-stops